generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -13,21 +14,30 @@
 
     public CheckpointOperation(State state) {
         super(state);
+        shortName = "Checkpoint";
     }
 
     @Override
     public void execute() {
         try {
+
+//            TODO: uncomment when verification supports it
+//            String cpStartRecord = String.format("%s, %s", shortName, "start");
+//            Correctness.recordOperation(cpStartRecord, false);
+
             MultiCheckpointWriter mcw = new MultiCheckpointWriter();
             mcw.addAllMaps(state.getMaps());
             long trimAddress = mcw.appendCheckpoints(state.getRuntime(), "Maithem");
             state.setTrimMark(trimAddress);
-            Thread.sleep(1000 * 30 * 1);
+            Thread.sleep(1000l * 30l * 1l);
             state.getRuntime().getAddressSpaceView().prefixTrim(trimAddress - 1);
             state.getRuntime().getAddressSpaceView().gc();
             state.getRuntime().getAddressSpaceView().invalidateClientCache();
             state.getRuntime().getAddressSpaceView().invalidateServerCaches();
-            System.out.println("CheckpointOperation Completed");
+
+//            TODO: uncomment when verification supports it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)